### PR TITLE
v3.1.0

### PR DESCRIPTION
**Pull request by bookchin**
* point to feature branch of core

### CODE REVIEW

**Comment by braydonf** [comment]
Tested with https://github.com/braydonf/storj-integration/tree/contact-in-offer

File: .jshintrc [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "undef": true,
   "unused": true,
   "maxparams": 4,
-  "maxstatements": 15,
+  "maxstatements": 18,
```
Comment (aleitner) [inline .jshintrc]: Bump this to 19

**Comment by braydonf** [comment]
Tested again https://github.com/braydonf/storj-integration/tree/contact-in-offer

File: bin/storjshare-logs.js [hunk 2]
```diff
@@ -88,6 +89,14 @@ utils.connectToDaemon(config.daemonRpcPort, function(rpc, sock) {
       }
     }
 
+    try {
+      if (fs.statSync(logFileDir).isDirectory()) {
```
Comment (aleitner) [inline bin/storjshare-logs.js]: Don't you need a `!` here?

**Pull request merged**